HW0/P4/P4.py

```diff
@@ -31,8 +31,12 @@
     # Normally, this data wouldn't be available in the real world
     #####################
 
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '--b', label='True trajectory')
+    inputs = np.loadtxt('P4_trajectory.txt', delimiter=',', usecols=(0,1,2))
```

Comment:
You can read the columns this way or using the pandas; I personally prefer pandas. It's worth knowing about!